intro-to-R-tidyverse/02-intro_to_ggplot2.Rmd

```diff
@@ -56,9 +56,14 @@ plots_dir <- "plots"
 
 We can also create a directory if it doesn't already exist.
 
+There's a couple ways that we can create that directory from within R.
+One way is to use the base R function `dir.create()`, which (as the name suggests) will create a directory.
+But, this function always assumes that the directory does not yet exist, and it will throw an error if you try to create an existing directory.
```

Comment:
Removed "always", as we can make it ignore the error with another option. And a bit of wording update
```suggestion
But this function assumes that the directory does not yet exist, and it will throw an error if you try to create a directory that already exists.
```